__tests__/add&remove.test.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @jest-environment jsdom
+*/
+import Add from '../src/modules/addTask';
+import deleteTask from '../src/modules/deleteTask';
+import getArr from '../src/modules/getDataFromLocalStorage';
+import saveInLocalStorage from '../src/modules/saveAtLocalStorage';
+import renderList from '../src/modules/renderList';
+import clearList from '../src/modules/clearList';
+
+describe('add & remove', () => {
+  // set up environment
+  document.body.innerHTML = `<main>+
+  <section id="task-list">+
+    <div id="list-title">+
+      <h2>Today's To Do</h2>+
+      <span><i class="fa-solid fa-arrows-rotate"></i></span>+
+    </div>+
+      <input id="task-text" type="text" placeholder="Add your task..." required>+
+      <button id="add-task-btn" type="submit"><i class="fa-solid fa-plus"></i></button>+
+    <div class="check-all">+
+      <input  type="checkbox">+
+      <label>SELECT ALL</label>+
+    </div>+
+    <ul id="list-placeholder"></ul>+
+    <button id="clear-btn">Clear all completed</button>+
+  </section>+
+</main>`;
+  document.getElementById('add-task-btn').onclick = Add;
+
+  // start add tests
+  describe('add task', () => {
+    test('only 1 li item is added in the DOM', () => {
+      const listPlaceholder = document.querySelector('#list-placeholder');
+      const listLengthBefore = listPlaceholder.children.length;
+      document.getElementById('task-text').value = 'fake task 1';
+      document.getElementById('add-task-btn').click();
+      const listLengthAfter = listPlaceholder.children.length;
```

Comment:
- [ ] You are almost there! However, both tests are failing, but if you run just one it passes, this is probably because a variable is being overwritten:

![Screen Shot 2022-07-22 at 7 15 40 PM](https://user-images.githubusercontent.com/83717285/180582696-b52d5c7c-6773-4c24-87e2-5711734cad60.png)

Check what's happening to this variable `const listLengthAfter ` and you will get some idea of what's causing the problem.

Reply:
Hey @smunozmo , thank a lot for your review and for pointing the direction on how to fix it. I'll will dig into that and fix this issue!

Reply:
@Tiago-Lelinski-Marin no problem, happy coding! 🤓